### DISCUSSION TITLE
Is using split panel with satellite imagery possible?

### DISCUSSION THREAD

**Discussion by osbm** [opengeos/leafmap#233] [Q&A] [answered]
I want to show the before & after satellite images on a fire ROI using the split panel. But all examples I can find are about basemaps. Is this possible?

**Comment by giswqs** [comment, marked as answer]
You can use `get_local_tile_layer()` to generate a tile layer from raster datasets. Then you can pass the tile layers to `left_layer` and `right_layer` of the `split_map()` function. 

https://leafmap.org/common/?h=get_local#leafmap.common.get_local_tile_layer
https://leafmap.org/notebooks/12_split_map/

**Reply by osbm** [reply]
I will try this out. You, sir are amazing :heart:
